api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -228,3 +254,53 @@ message Pass {
   // this pass.
   double uplink_center_frequency_hz = 10;
 }
+
+// A scheduled plan, specifying a time range within which a satellite will be communicated with.
+message Plan {
+  // The unique ID of the plan.
+  string id = 1;
+
+  // The id of the satellite to be tracked in the plan.
+  string satellite_id = 2;
+
+  // A description of the status of a plan.
+  enum Status {
+    // The plan is reserved for execution in the future at `aos_time`.
+    RESERVED = 0;
+
+    // The plan is currently executing and will end in the future at `los_time`.
+    EXECUTING = 1;
+
+    // The plan already finished executing successfully.
+    SUCCEEDED = 2;
+
+    // The plan failed to execute correctly.
+    FAILED = 3;
+
+    // The plan was canceled. This can be explicitly by calling `CancelPlan` or implicit due to
+    // cancellation by the ground station in the plan (i.e., due to bad weather).
+    CANCELED = 4;
+  }
+  // The status of this plan.
+  Status status = 3;
+
+  // The time of Acquisition of Signal (AOS) between the ground station and satellite in this plan.
+  google.protobuf.Timestamp aos_time = 4;
+
+  // The time of Loss of Signal (LOS) between the ground station and satellite in this plan.
+  google.protobuf.Timestamp los_time = 5;
+
+  // The max elevation during the plan, in degrees.
+  double max_elevation_degrees = 6;
+
+  // The time of the max elevation during the plan.
+  google.protobuf.Timestamp max_elevation_time = 7;
+
+  // The center frequency, in Hz, for downlinking in this plan. 0 if downlink is not available in
+  // this plan.
+  double downlink_center_frequency_hz = 8;
```

Comment:
uint64 
Better to be consistent with RadioDeviceConfiguration.center_frequency_hz